translation-language-detection-api-playground/script.js

```diff
@@ -6,7 +6,7 @@
 (async () => {
   // The Language Detector API uses the `self.LanguageDetector` namespace in Canary, but
   // `ai.languageDetector` in Stable, so both shapes are feature detected.
-  if (!('LanguageDetector' in self) && !('ai' in self) && !('languageDetector' in self)) {
```

Comment:
Note that Chrome stable uses the new API shape now, so we could just drop the `self.ai` detection?